src/js/common/schemaform/reducers/index.js

```diff
@@ -104,6 +104,7 @@ export default function createSchemaFormReducer(formConfig) {
       loadedStatus: LOAD_STATUSES.notAttempted,
       version: formConfig.version,
       formId: formConfig.formId,
+      lastSaveDate: null,
```

Comment:
Yeah, that's alright. The only time it's used is in that save page which, as you said, is distinct from the loadedData. The loaded data will have `metadata.savedAt` for when it was saved (to be displayed on the intro page) and they'll be _basically_ the same thing, but....not.